.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
Is this file required?

Reply:
Hello @tkuhrt  - apologies, I put in a .gitignore to ignore these files - it clearly did not work :(
I'll fix it and re-submit